Update Data Table (#26)
Rensvandeschoot/software-overview-machine-learning-for-screening-text: README.md

|      [FASTREAD](#fastread)      |                    CSV                    |                                   |                             |                             |                             |                             |
|        [Rayyan](#rayyan)        |    RIS, ENW, BIB, CSV, XML, CIW, NBIB     |         :zap:<sup>3</sup>         |     :white_check_mark:      |             :x:             |             :x:             |     :white_check_mark:      |
|  [RobotAnalyst](#robotanalyst)  |                                           |                                   |                             |                             |                             |                             |
|  [SWIFT-Review](#swift-review)  |                                           |                                   |                             |                             |                             |                             |

:white_check_mark: Yes/Implemented;
:x: No/Not implemented;
:zap: Only for some extensions (add a footnote for more explanation);
:grey_question: Unknown (requires an issue).

> <sup>1</sup> ASReview provides several [data tools](https://github.com/asreview/asreview-datatools) to convert several file formats,

Reviewer: ```suggestion
<sup>1</sup> ASReview provides several open-source [tools](https://github.com/asreview/asreview-datatools) to convert file formats (e.g., CSV->RIS, or RIS->XLSX),
```

> and to combine multiple (labeled, partly labeled, or unlabeled) datasets.

Reviewer: ```suggestion
to combine datasets (labeled, partly labeled, or unlabeled) and deduplication.
```

<sup>2</sup> List of PubMed IDs

> <sup>3</sup> It is possible to recreate a partly labeled dataset by adding all records of the same label (relevant/irrelevant/unlabeled) as separate files.

Reviewer: ```suggestion
<sup>3</sup> It is possible to upload all records of the same label (relevant/irrelevant/unlabeled) as separate files.
```

> <sup>4</sup> EPPI-Reviewer provides an [online file converter](https://eppi.ioe.ac.uk/cms/Default.aspx?tabid=2934) to convert several file formats to RIS.

Reviewer: ```suggestion
<sup>4</sup> EPPI-Reviewer provides a closed-source [online file converter](https://eppi.ioe.ac.uk/cms/Default.aspx?tabid=2934) to convert several file formats to RIS.
```
